spec/lib/money/bank/russian_central_bank_spec.rb

```diff
@@ -109,4 +109,14 @@
       end
     end
   end
+
+  describe '#indirect_rate' do
+    before do
+      bank.flush_rates
+    end
+
+    it 'should return nil when indirect rate cannot be calculated' do
+      expect(bank.send(:indirect_rate, 'RUB', 'RUB')).to be_nil
```

Comment:
@olhor Curious to know how this test is successful.

With the current gem version (1.1.3) tried - 

```
[24] pry(main)> obj.send(:indirect_rate, 'RUB', 'RUB')
=> 1.0
```

But it gives `stack level too deep` error on UAE currency.

```
[23] pry(main)> obj.send(:indirect_rate, 'AED', 'RUB')
SystemStackError: stack level too deep
```

IMO the UAE currency would be the right example to know if the fix is working.

Reply:
Before the test I call the `flush_rates` method. In your first example it looks like RUB rates were still present. With the fix I get:
```
[3] pry(main)> bank = Money::Bank::RussianCentralBank.new
[4] pry(main)> bank.send(:indirect_rate, 'RUB', 'RUB')
=> nil
[5] pry(main)> bank.send(:indirect_rate, 'PLN', 'UAH')
=> nil
```
So it works fine for me.

Reply:
@olhor Ok. Can you please try with `AED` currency.

Thanks.

Reply:
```
[6] pry(main)> bank.send(:indirect_rate, 'AED', 'AED')
=> nil
[7] pry(main)> bank.send(:indirect_rate, 'PLN', 'AED')
=> nil
[8] pry(main)> bank.send(:indirect_rate, 'AED', 'PLN')
=> nil
```
Should I additional expectations to the test?